src/library/scala/collection/mutable/PriorityQueue.scala

```diff
@@ -15,6 +15,12 @@ import generic._
 /** This class implements priority queues using a heap.
  *  To prioritize elements of type A there must be an implicit
  *  Ordering[A] available at creation.
+ * 
+ *  If multiple elements have the same priority in the ordering of this
+ *  PriorityQueue, no guarantees are made regarding the order in which elements
+ *  are returned by `dequeue` or `dequeueAll`. In particular, that means this
+ *  class does not guarantee first in first out behaviour that nay be
```

Comment:
typo: "that *nay* be" should be "that *may* be"

Reply:
I fixed that! I just didn't notice I didn't push it. I'm not sure whether that makes it better or worse.